src/OrchardCore.Modules/OrchardCore.ContentFields/Migrations.cs

```diff
@@ -0,0 +1,60 @@
+using OrchardCore.ContentFields.Fields;
+using OrchardCore.ContentFields.Settings;
+using OrchardCore.ContentManagement.Metadata;
+using OrchardCore.Data.Migration;
+
+namespace OrchardCore.ContentFields
+{
+    public class Migrations : DataMigration
+    {
+        private readonly IContentDefinitionManager _contentDefinitionManager;
+
+        public Migrations(IContentDefinitionManager contentDefinitionManager)
+        {
+            _contentDefinitionManager = contentDefinitionManager;
+        }
+
+        // This migration does not need to run on new installations, but because there is no
+        // initial migration record, there is no way to shortcut the Create migration.
+        public int Create()
+        {
```

Comment:
I understand. At some point I thought about adding a default entry in the migrations, such that if a module is enabled at least once then it would have an entry. Like a default Create().